test/integration/component/test_ldap.py

```diff
@@ -21,13 +21,18 @@
 
 #!/usr/bin/env python
 from marvin.cloudstackTestCase import cloudstackTestCase
+#from marvin.cloudstackTestCase import *
```

Comment:
Please remove code, do not comment it. It can be retrieved if necessary from Git.